.github/workflows/ci.yml

```diff
@@ -40,3 +40,20 @@ jobs:
         uses: docker://php:7.3-cli
         with:
           args: ./tools/psalm --config=.psalm/config.xml --no-progress --shepherd --show-info=false --stats
+
+  backward-compatibility:
+    name: Backward Compatibility
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Build Docker image for PHP 7.3
```

Comment:
I've added this as a separate (!) step so it's more obvious how much time is consumed for building the Docker image for PHP 7.3.